docs/plugins/keeweb-mojave/plugin.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * KeeWeb plugin: keeweb-mojave
+ * @author agurodriguez
+ * @license MIT
+ */
+
+const AppSettingsModel = require('models/app-settings-model');
+const FeatureDetector = require('util/feature-detector');
+
+const FIRST_RUN_KEY = 'keeweb-mojave-first-run';
+const PREVIOUS_TITLEBAR_STYLE_KEY = 'keeweb-mojave-previous-titlebar-style';
+
+if (FeatureDetector.isDesktop) {
+    if (!localStorage.getItem(FIRST_RUN_KEY)) {
```

Comment:
localStorage will not be persisted on desktop, you can use runtime-data module for this